IT-Infrastructure.md

```diff
@@ -0,0 +1,29 @@
+FORT is designed to allow a remote monitoring of multiple experiments running in parallel, to save space in the experiment room and at the same time offer a simple and modular hardware setup. The resulting IT infrastructure is explained in the chart below.
```

Comment:
Due to the very nature of the tracking system, having to deal with large amount of binary data, distributed in real time between several computers that have to perform online computation, the IT infrastructure has been designed accros modular micro services To simplify the development and ease the use of the system, the philosophy of the infrastructure is to have low to none security in any of the standard service (SSH,VNC) or application specific service (Leto,Zeus). __**Therefore it is advised to run the FORT Setup in a strictly private Local Area Network behind a propperly  configured firewall**__. 

For the end user it means that any operation (configuration, starting and stopping experiment) on the system has to be performed on site and cannot be operated remotely. A specific webapp however has been designed to be able to monitor remotely the setup via a webserver, but doesn't allow any control of the system, by design.